src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -615,7 +817,7 @@ const FormCreationForm = ({csrftoken, formUuid, formHistoryUrl }) => {
                 : null
             }
 
-            <Tabs>
+            <Tabs defaultIndex={5}>
```

Comment:
```suggestion
            <Tabs>
```
was only needed for development/debugging